WooCommerce/WooCommerceTests/ViewRelated/CardPresentPayments/CardPresentPaymentsOnboardingUseCaseTests.swift

```diff
@@ -369,7 +375,11 @@ class CardPresentPaymentsOnboardingUseCaseTests: XCTestCase {
         _ = CardPresentPaymentsOnboardingUseCase(storageManager: storageManager, stores: stores)
 
         // Then
-        XCTAssertEqual(stores.receivedActions.count, 2)
+
+        // AppSettingsAction.loadStripeInPersonPaymentsSwitchState
+        // + AppSettingsAction.loadCanadaInPersonPaymentsSwitchState
+        // + CardPresentPaymentAction.use
+        XCTAssertEqual(stores.receivedActions.count, 3)
         let action = try XCTUnwrap(stores.receivedActions.last as? CardPresentPaymentAction)
```

Comment:
These tests do seem a bit brittle and annoying don't they... Perhaps we can do this instead:
```suggestion
        let action = stores.receivedActions.first(where: { $0 is CardPresentPaymentAction }) as? CardPresentPaymentAction
```

I've tried this locally and it works

Reply:
I guess it depends on what we want to test exactly and both options make some assumptions. I think I prefer the existing one because the last action should be `use`. If we wanted to allow for more flexibility, we'd still want to get the last `use` not the first. If I hadn't done the optimization where `updateState` is only called once on initialization, we'd be getting two `use` actions if the store had both features enabled.

Reply:
> Only other thing - did you consider removing the feature flag and instead checking for `.localDeveloper || .alpha` where we show the experimental toggle? It could help to avoid confusion/accidental use of the Feature flag rather than the toggle to gate features. I don't mind but just a thought 😊

I think that's a valid concern, but I'm not sure I want to start using `BuildConfiguration` values outside the feature flag service. Maybe the ideal solution would be a system that takes the two concepts (feature flag and experimental feature toggles), and makes them work more cohesively. Though it feels hard to justify investing time on it, since it's code that will end up being deleted in the short term.

Reply:
> the last action should be `use`. If we wanted to allow for more flexibility, we'd still want to get the last `use` not the first. If I hadn't done the optimization where `updateState` is only called once on initialization, we'd be getting two `use` actions if the store had both features enabled.

Makes sense... perhaps just worth removing the assert on the count of actions then?

Reply:
> I'm not sure I want to start using `BuildConfiguration` values outside the feature flag service. 

Fair enough... instead of larger rework, perhaps we should a shared feature flag called `shouldShowAlphaExperimentalToggles`, which is set to `.localDeveloper || .alpha`, and can be used for multiple toggles in the experimental features screen?

Reply:
That would make a lot of sense I think. I think a long time ago `BuildConfiguration` supported beta and internal builds, so there were a few more possibilities, but these days we seem to always toggle between dev+alpha only and including production, so maybe we can use a simpler system 🤔 

Out of scope for this PR, but I'd be up for a proposal like that

Reply:
> Makes sense... perhaps just worth removing the assert on the count of actions then?

Sorry, I saw this last comment after hitting the merge button, and also realized I nested my comments in the wrong place 🤦🏽

Reply:
Removed those checks in #6046